src/common/constants.js

```diff
@@ -123,3 +123,26 @@ export const TRANSACTION = {
 export const TIMEOUT_VALUE = {
   WALLET_CONNECT: 20000, // 20 sec
 };
+
+export const WEAK_PASSCODES = [
```

Comment:
can you order? like: 0000, 1111, ...?

Reply:
It's sorted by popularity or probability of usage according to [PIN number analysis](https://www.datagenetics.com/blog/september32012/) do you still want to sort it alphabetically?

Reply:
Nice, would it be nice to leave the link source as a comment then, wouldn’t it?

Reply:
Sure, added 👍